fiftyone/utils/huggingface.py

```diff
@@ -63,6 +64,26 @@
 
 logger = logging.getLogger(__name__)
 
+def list_hub_datasets():
```

Comment:
I recommend this method work similarly `fo.list_datasets()` where the method returns the names and the user can print them themselves if that's what they want to do. Or they do something programmatically with the names